pandas/tests/frame/test_stack_unstack.py

```diff
@@ -896,8 +902,13 @@ def verify(df):
         )
 
         right = DataFrame(vals, columns=cols, index=idx)
+        if using_array_manager:
+            # INFO(ArrayManager) with ArrayManager preserve dtype where possible
```

Comment:
is `INFO(ArrayManager)` a pattern i should know?

Reply:
Yeah, it was requested by Jeff on one of the previous PRs. It's an explicit comment about behaviour that changed with ArrayManager, but which is not a TODO (since it's not wrong behaviour that still needs to be fixed)

Reply:
thanks